pom.xml

```diff
@@ -61,7 +61,7 @@
     <properties>
         <check.skip-dependency-versions>true</check.skip-dependency-versions>
         <check.spotbugs-exclude-filter-file>${main.basedir}/spotbugs-exclude.xml</check.spotbugs-exclude-filter-file>
-        <killbill-api.version>0.54.0-9b20b32-SNAPSHOT</killbill-api.version>
+        <killbill-api.version>0.54.0-7a22bc7-SNAPSHOT</killbill-api.version>
         <killbill-client.version>1.3.0-a6e0394-SNAPSHOT</killbill-client.version>
```

Comment:
Could you please specify the `killbill-api` version as `0.54.0-e27330f-SNAPSHOT` (latest version of `killbill-api`)?

Reply:
I merged your PR and as part of the merge got those versions updated.